[2024-01-21T13:38:52.859+0000] {processor.py:161} INFO - Started process (PID=36) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:38:52.869+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:38:52.907+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:38:52.904+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:38:53.305+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:38:53.217+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:38:53.337+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:38:53.858+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 1.282 seconds
[2024-01-21T13:39:42.569+0000] {processor.py:161} INFO - Started process (PID=97) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:39:42.572+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:39:42.577+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:39:42.576+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:39:42.600+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:39:42.596+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:39:42.602+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:39:42.681+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.118 seconds
[2024-01-21T13:40:18.881+0000] {processor.py:161} INFO - Started process (PID=142) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:40:18.935+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:40:18.980+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:40:18.979+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:40:19.059+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:40:19.048+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:40:19.063+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:40:19.208+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.345 seconds
[2024-01-21T13:40:49.317+0000] {processor.py:161} INFO - Started process (PID=198) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:40:49.319+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:40:49.322+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:40:49.322+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:40:49.338+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:40:49.336+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:40:49.340+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:40:49.383+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.074 seconds
[2024-01-21T13:41:19.486+0000] {processor.py:161} INFO - Started process (PID=251) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:41:19.489+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:41:19.495+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:41:19.494+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:41:19.512+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:41:19.508+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:41:19.513+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:41:19.553+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.076 seconds
[2024-01-21T13:41:50.127+0000] {processor.py:161} INFO - Started process (PID=306) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:41:50.192+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:41:50.210+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:41:50.209+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:41:50.266+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:41:50.239+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:41:50.273+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:41:50.378+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.259 seconds
[2024-01-21T13:42:20.734+0000] {processor.py:161} INFO - Started process (PID=361) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:42:20.751+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:42:20.768+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:42:20.767+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:42:20.793+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:42:20.788+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:42:20.796+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:42:20.871+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.154 seconds
[2024-01-21T13:42:51.319+0000] {processor.py:161} INFO - Started process (PID=416) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:42:51.322+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:42:51.329+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:42:51.328+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:42:51.342+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:42:51.339+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:42:51.345+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:42:51.389+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.077 seconds
[2024-01-21T13:43:21.572+0000] {processor.py:161} INFO - Started process (PID=471) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:43:21.575+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:43:21.579+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:43:21.578+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:43:21.594+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:43:21.592+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:43:21.596+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:43:21.632+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.067 seconds
[2024-01-21T13:43:51.792+0000] {processor.py:161} INFO - Started process (PID=527) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:43:51.794+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:43:51.799+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:43:51.798+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:43:51.827+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:43:51.820+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:43:51.829+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:43:51.883+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.098 seconds
[2024-01-21T13:44:22.275+0000] {processor.py:161} INFO - Started process (PID=581) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:44:22.282+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:44:22.289+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:44:22.289+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:44:22.347+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:44:22.345+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:44:22.348+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:44:22.383+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.122 seconds
[2024-01-21T13:44:52.942+0000] {processor.py:161} INFO - Started process (PID=637) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:44:52.949+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:44:52.959+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:44:52.959+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:44:53.016+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:44:53.012+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:44:53.021+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:44:53.089+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.154 seconds
[2024-01-21T13:45:23.269+0000] {processor.py:161} INFO - Started process (PID=693) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:45:23.271+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:45:23.275+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:45:23.274+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:45:23.286+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:45:23.284+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:45:23.288+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:45:23.322+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.059 seconds
[2024-01-21T13:45:53.695+0000] {processor.py:161} INFO - Started process (PID=749) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:45:53.697+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:45:53.700+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:45:53.700+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:45:53.711+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:45:53.709+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:45:53.712+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:45:53.744+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.054 seconds
[2024-01-21T13:46:25.323+0000] {processor.py:161} INFO - Started process (PID=805) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:46:25.344+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:46:25.381+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:46:25.380+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:46:25.466+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:46:25.462+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:46:25.478+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:46:25.685+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.567 seconds
[2024-01-21T13:46:55.895+0000] {processor.py:161} INFO - Started process (PID=860) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:46:55.897+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:46:55.901+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:46:55.901+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:46:55.911+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:46:55.909+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:46:55.912+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:46:55.948+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.061 seconds
[2024-01-21T13:47:26.459+0000] {processor.py:161} INFO - Started process (PID=916) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:47:26.462+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:47:26.467+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:47:26.467+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:47:26.483+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:47:26.481+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:47:26.485+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:47:26.586+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.136 seconds
[2024-01-21T13:47:56.890+0000] {processor.py:161} INFO - Started process (PID=972) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:47:56.893+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:47:56.900+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:47:56.899+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:47:56.921+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:47:56.917+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:47:56.923+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:47:56.976+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.094 seconds
[2024-01-21T13:48:27.585+0000] {processor.py:161} INFO - Started process (PID=1028) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:48:27.587+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:48:27.591+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:48:27.591+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:48:27.605+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:48:27.602+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:48:27.607+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:48:27.642+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.063 seconds
[2024-01-21T13:48:58.639+0000] {processor.py:161} INFO - Started process (PID=1084) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:48:58.645+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:48:58.652+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:48:58.651+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:48:58.679+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:48:58.672+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:48:58.681+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:48:58.819+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.190 seconds
[2024-01-21T13:49:28.922+0000] {processor.py:161} INFO - Started process (PID=1140) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:49:28.926+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:49:28.931+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:49:28.930+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:49:28.947+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:49:28.944+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:49:28.948+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:49:28.992+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.078 seconds
[2024-01-21T13:49:59.309+0000] {processor.py:161} INFO - Started process (PID=1197) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:49:59.314+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:49:59.328+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:49:59.322+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:49:59.369+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:49:59.363+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:49:59.371+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:49:59.532+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.232 seconds
[2024-01-21T13:50:29.922+0000] {processor.py:161} INFO - Started process (PID=1251) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:50:29.924+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:50:29.929+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:50:29.928+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:50:29.945+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:50:29.941+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:50:29.947+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:50:29.989+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.074 seconds
[2024-01-21T13:51:00.752+0000] {processor.py:161} INFO - Started process (PID=1307) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:51:00.754+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:51:00.758+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:51:00.757+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:51:00.768+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:51:00.766+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:51:00.770+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:51:00.806+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.059 seconds
[2024-01-21T13:51:31.263+0000] {processor.py:161} INFO - Started process (PID=1363) to work on /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:51:31.265+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/wikipedia_flow.py for tasks to queue
[2024-01-21T13:51:31.269+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:51:31.269+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:51:31.281+0000] {logging_mixin.py:188} INFO - [2024-01-21T13:51:31.279+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/wikipedia_flow.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/wikipedia_flow.py", line 10, in <module>
    from pipelines.wikipedia_pipeline import extract_wikipedia_data, transform_wikipedia_data, write_wikipedia_data
ModuleNotFoundError: No module named 'pipelines'
[2024-01-21T13:51:31.282+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/wikipedia_flow.py
[2024-01-21T13:51:31.325+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/wikipedia_flow.py took 0.068 seconds
